R/rnet_junctions.R

```diff
@@ -0,0 +1,30 @@
+#' Get points where 3 or more linestrings meet (junctions on road networks)
```

Comment:
See below for some comments on the name (and objective) of the function.

Reply:
You're 100% right, as you say and as shown in the reprex above. I think `rnet_junction()` is still leaded. You happy to review a PR that eventually implements that idea (currently only in the `trafficalmr` pkg)?

Reply:
> I think rnet_junction() is still leaded. 

I'm not sure what you mean 😅 

>  You happy to review a PR that eventually implements that idea (currently only in the trafficalmr pkg)?

Sure! I didn't read the discussion in trafficalmr (but I will) but I would implement it as described in the previous comment.